<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tableWidget</class>
 <widget class="QWidget" name="tableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{ border: none }
#tableWidget { border: 9px solid rgb(40, 44, 52); border-radius: 18px }
#contentTop .QLabel { background-color: transparent; font-size: 15px }
QWidget {
	background-color: rgb(40, 44, 52);
	color: rgb(221, 221, 221);
	font: 10pt &quot;Segoe UI&quot; }
QPushButton {
	border: 3px solid rgb(52, 59, 72);
	border-radius: 5px;	
	background-color: rgb(52, 59, 72);
	font-size: 15px; padding: 1px }
QPushButton:hover { background-color: rgb(57, 65, 80) }
QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 3px solid rgb(43, 50, 61) }
QLineEdit {
	background-color: rgb(33, 37, 43);
	border-radius: 5px;
	border: 2px solid rgb(33, 37, 43);
	padding-left: 3px;
	font-size: 15px;
	selection-color: rgb(255, 255, 255);
	selection-background-color: rgb(255, 121, 198) }
QLineEdit:hover { border: 2px solid rgb(64, 71, 88) }
QLineEdit:focus { border: 2px solid rgb(91, 101, 124) }
QCheckBox::indicator {
    border: 3px solid rgb(52, 59, 72);
	width: 15px;
	height: 15px;
	border-radius: 10px;
    background: rgb(44, 49, 60) }
QCheckBox::indicator:hover { border: 3px solid rgb(58, 66, 81) }
QCheckBox::indicator:checked {
    background: 3px solid rgb(52, 59, 72);
	border: 3px solid rgb(52, 59, 72);
	background-image: url(:/icons/icons/check.svg) }
QScrollBar:horizontal {
	border: none;
	background: rgb(52, 59, 72);
	height: 8px;
	margin: 0px 21px 0 21px;
	border-radius: 0px }
QScrollBar::handle:horizontal {
	background: rgb(189, 147, 249);
	min-width: 25px;
	border-radius: 4px }
QScrollBar::handle:horizontal:hover, QScrollBar::handle:vertical:hover {
	background: rgb(208, 181, 249) }
QScrollBar::handle:horizontal:pressed, QScrollBar::handle:vertical:pressed {
	background: rgb(161, 103, 249) }
QScrollBar::add-line:horizontal {
	border: none;
	background: rgb(55, 63, 77);
	width: 20px;
	border-top-right-radius: 4px;
	border-bottom-right-radius: 4px;
	subcontrol-position: right;
	subcontrol-origin: margin }
QScrollBar::sub-line:horizontal {
	border: none;
	background: rgb(55, 63, 77);
	width: 20px;
	border-top-left-radius: 4px;
	border-bottom-left-radius: 4px;
	subcontrol-position: left;
	subcontrol-origin: margin }
QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal,
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal,
QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical,
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none }
 QScrollBar:vertical {
	border: none;
    background: rgb(52, 59, 72);
    width: 8px;
    margin: 21px 0 21px 0;
	border-radius: 0px }
QScrollBar::handle:vertical {	
	background: rgb(189, 147, 249);
	min-height: 25px;
	border-radius: 4px }
QScrollBar::add-line:vertical {
	border: none;
	background: rgb(55, 63, 77);
	height: 20px;
	border-bottom-left-radius: 4px;
	border-bottom-right-radius: 4px;
	subcontrol-position: bottom;
	subcontrol-origin: margin }
QScrollBar::sub-line:vertical {
	border: none;
	background: rgb(55, 63, 77);
	height: 20px;
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
	subcontrol-position: top;
	subcontrol-origin: margin }
QScrollBar::add-line:horizontal:hover, QScrollBar::sub-line:horizontal:hover,
QScrollBar::add-line:vertical:hover, QScrollBar::sub-line:vertical:hover {
	background: rgb(64, 69, 77) }
QScrollBar::add-line:horizontal:pressed, QScrollBar::sub-line:horizontal:pressed,
QScrollBar::add-line:vertical:pressed, QScrollBar::sub-line:vertical:pressed {
	background: rgb(189, 147, 249) }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="contentTop">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_run">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_add">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>ADD ROW</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_delete">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>DELETE ROW</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tableImport">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string> IMPORT</string>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/icons/icons/text2table.svg</normaloff>:/icons/icons/text2table.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tableExport">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string> EXPORT</string>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/icons/icons/table2text.svg</normaloff>:/icons/icons/table2text.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTop">
       <widget class="QPushButton" name="closeBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(65, 74, 90);
	border: 4px solid rgb(65, 74, 90);
	border-radius: 10px; }
QPushButton:hover { background-color: rgb(52, 59, 72); border: 4px solid rgb(52, 59, 72) }
QPushButton:pressed { background-color: rgb(23, 26, 30); border: 4px solid rgb(23, 26, 30) }</string>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/icons/icons/close.svg</normaloff>:/icons/icons/close.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>10</width>
          <height>12</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignmentFlag::AlignTop">
    <widget class="QFrame" name="latestPost">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="latestPostCheckBox">
        <property name="text">
         <string>Lấy post mới nhất</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="postIndex">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Điền số thứ tự post trong group</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
    <widget class="QFrame" name="filterGroup">
     <layout class="QHBoxLayout" name="_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="filterGroupCheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Filter</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filterGroupInput">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>vps, proxy, rdp</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignmentFlag::AlignTop">
    <widget class="QFrame" name="fromToFrame">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="listFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="listLabel">
           <property name="text">
            <string>List post</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="listBox">
        <property name="placeholderText">
         <string>ngăn cách nhau bới dấu ,</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="useFromToFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="fromLabel">
           <property name="text">
            <string>Từ post</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fromBox">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="toLabel">
           <property name="text">
            <string>đến post</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="toBox">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_fromTo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/icons/icons/change.svg</normaloff>:/icons/icons/change.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="statusTable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="countRows">
       <property name="text">
        <string>Total rows:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableCornerButton::section { background-color: rgb(33, 37, 43) }
QTableWidget {	
	padding: 5px;
	gridline-color: rgb(44, 49, 58);
	border-bottom: 1px solid rgb(44, 49, 60); }
QTableWidget::item{ border-color: rgb(44, 49, 60) }
QTableWidget::item:selected{
	background-color: rgb(189, 147, 249);
	color: rgb(40, 44, 52);
}
QHeaderView { qproperty-defaultAlignment: AlignCenter }
QHeaderView::section{
	background-color: rgb(33, 37, 43);
	border: 1px solid rgb(44, 49, 60);
	font-size: 15px }
QLineEdit {
    background-color: rgb(50, 54, 62); /* slightly lighter/darker variant for edit mode */
	selection-background-color: rgb(189, 147, 249); /* background when highlight */
    selection-color: rgb(40, 44, 52); /* text color when selected */
}</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>110</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <column>
      <property name="text">
       <string>LINK GROUP</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>LINK POST</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NAME GROUP</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>STATUS</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
